contracts/HATVaults.sol

```diff
@@ -1118,7 +1129,7 @@ contract  HATVaults is Governable, ReentrancyGuard {
         }
     }
 
-    function pendingReward(uint256 _pid, address _user) external view returns (uint256) {
+    function pendingReward(uint256 _pid, address _user) public view returns (uint256) {
         PoolInfo storage pool = poolInfos[_pid];
         UserInfo storage user = userInfo[_pid][_user];
         uint256 rewardPerShare = pool.rewardPerShare;
```

Comment:
@jellegerbrandy why not update the pool's reward per share if we enter the next if block?
we calculate already.. can update lastRewardBlock so it will not update next

Reply:
actually what is this used for? if it's only for the user to check for himself then this should use msg.sender, and ok to not update

Reply:
so I guess this is a convenience function for the ui